Examples/IO/XML/itkParticleSwarmOptimizerSAXWriter.cxx

```diff
@@ -97,7 +97,7 @@ ParticleSwarmOptimizerSAXWriter::WriteFile()
     ofs << " value=\"";
     for (const auto & bound : this->m_InputObject->GetParameterBounds())
     {
-      ofs << " " << bound.first;
+      ofs << ' ' << bound.first;
```

Comment:
This was somehow missed by e8773472ddc1cd3bb0a6b638e1cd70a6b4616160?

Reply:
@dzenanz Indeed, well observed 👌 !  Commit e8773472ddc1cd3bb0a6b638e1cd70a6b4616160 only did the Modules, while the current PR does include the Examples dir.